Formula/man-db.rb

```diff
@@ -0,0 +1,45 @@
+class ManDb < Formula
+  desc "Modern, featureful implementation of the Unix man page system"
+  homepage "https://nongnu.org/man-db/"
+  url "https://download.savannah.nongnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  depends_on "libpipeline"
+
+  def install
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-systemdtmpfilesdir=no",
+                          "--with-systemdsystemunitdir=no",
+                          "--disable-cache-owner",
+                          "--disable-setuid"
+    ENV.append_to_cflags "-Wl,-flat_namespace,-undefined,suppress"
+    system "make", "CFLAGS=#{ENV.cflags}"
```

Comment:
Remove `CFLAGS` which are already in the environment anyway.

Reply:
This was at @lembacon's suggestion. See https://github.com/Homebrew/homebrew-core/pull/36469#discussion_r251331095.